### DISCUSSION TITLE
Support for Spark 3.3.0 and Databricks 11 Runtime

### DISCUSSION THREAD

**Discussion by james-miles-ccy** [nightscape/spark-excel#674] [General]
Hi there, just wondering when the latest version Spark 3.3 will be supported by Spark Excel? 

I am currently using the latest runtime 11 in Databricks which runs on Spark 3.3.0 and Spark-Excel is no longer functioning correctly.

**Comment by nightscape** [comment]
I just added Spark 3.3.0 to the build matrix, but [it's failing](https://github.com/crealytics/spark-excel/actions/runs/3369856567/jobs/5590029278).
If anybody is willing to create a PR, I'd be happy to merge.

**Reply by coryvisi** [reply]
Can you add the failure logs to the discussion please? The logs have expired from the build report.

**Comment by coryvisi** [comment]
AWS Glue v4 also runs on Spark 3.3.0. Support for this version would be helpful for many users.

**Reply by nightscape** [reply]
Did you see https://github.com/crealytics/spark-excel/pull/679 ?
spark-excel should support Spark 3.3.

**Reply by coryvisi** [reply]
Will it now build successfully on Spark 3.3.0, specifically?

**Reply by nightscape** [reply]
Yes, see eg https://github.com/crealytics/spark-excel/actions/runs/5232155930/jobs/9446726677
